packages/website/index.html

```diff
@@ -17,42 +17,19 @@
     />
     <script src="https://cdn.jsdelivr.net/npm/vanillajs-datepicker@1.3.4/dist/js/datepicker-full.min.js"></script>
 
-    <link rel="icon" href="./favicon.ico" type="image/x-icon" />
-    <link rel="icon" type="image/png" sizes="16x16" href="./favicon16x16.png" />
-    <link rel="icon" type="image/png" sizes="32x32" href="./favicon32x32.png" />
+    <link rel="icon" href="/favicon.ico" type="image/x-icon" />
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon16x16.png" />
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon32x32.png" />
     <link
       rel="icon"
       type="image/png"
       sizes="192x192"
-      href="./favicon192x192.png"
+      href="/favicon192x192.png"
     />
-    <script type="text/javascript">
-      // Single Page Apps for GitHub Pages
-      // MIT License
-      // https://github.com/rafgraph/spa-github-pages
-      // This script checks to see if a redirect is present in the query string,
-      // converts it back into the correct url and adds it to the
-      // browser's history using window.history.replaceState(...),
-      // which won't cause the browser to attempt to load the new url.
-      // When the single page app is loaded further down in this file,
-      // the correct url will be waiting in the browser's history for
-      // the single page app to route accordingly.
-      (function(l) {
-        if (l.search[1] === '/' ) {
-          var decoded = l.search.slice(1).split('&').map(function(s) {
-            return s.replace(/~and~/g, '&')
-          }).join('?');
-          window.history.replaceState(null, null,
-              l.pathname.slice(0, -1) + decoded + l.hash
-          );
-        }
-      }(window.location))
-    </script>
-
-    <script type="module" src="/src/main.tsx"></script>
   </head>
   <body>
-    <div id="app" class="flex min-h-full bg-white"></div>
+    <div id="app" class="flex min-h-full bg-white"><!--app-html--></div>
     <div id="portal"></div>
+    <script type="module" src="/src/entry-client.tsx"></script>
```

Comment:
this is the equivalent to the old `main.tsx`. We call it `client` following convention from the react SSR docs